src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculator {
```

Comment:
Name should be more specific. `Calculator` calculates everything, but you calculate only `area`.